pkg/server/api/svid/v1/server.go

```diff
@@ -54,8 +55,39 @@ func (s server) MintX509SVID(ctx context.Context, req *svid.MintX509SVIDRequest)
 	}, nil
 }
 
-func (s server) MintJWTSVID(context.Context, *svid.MintJWTSVIDRequest) (*svid.MintJWTSVIDResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "not implemented")
+func (s server) MintJWTSVID(ctx context.Context, req *svid.MintJWTSVIDRequest) (*svid.MintJWTSVIDResponse, error) {
+	log := rpccontext.Logger(ctx)
+
+	if req.Id == nil {
+		log.Error("Request must specify SPIFFE ID")
+		return nil, status.Error(codes.InvalidArgument, "request must specify SPIFFE ID")
+	}
+	spiffeID, err := spiffeid.New(req.Id.TrustDomain, req.Id.Path)
+	if err != nil {
+		log.WithError(err).Error("Failed to parse SPIFFE ID")
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	if len(req.Audience) == 0 {
```

Comment:
This validation is being done in the service. I don't think we need to repeat it here.